source/includes/agents/_30-update.md.erb

```diff
@@ -77,7 +77,7 @@ Atleast one of the following properties must be specified, not specifying the pr
   describe_object(nil) do
     hostname              'String',             'The new hostname'
     resources             ['String', 'Array'],  'A comma separated strings of resources, or an array of string resources. Updating of resources is **NOT** supported by elastic agents.'
-    environments          ['String', 'Array'],  'A comma separated strings of environments, or an array of string environments'
+    environments          ['String', 'Array'],  'A comma separated strings of environments, or an array of string environments. If the given enviornment is present, the agent will be mapped to it.'
```

Comment:
How about,
`A comma separated strings of environments, or an array of string environments. The agent will be associated to an environment only if an environment exists in GoCD for the given name, else will be ignored.`